### PR TITLE
Logout/Delete Account fully worked.

### PR DESCRIPTION
**Pull request by RomanWSU**
Log Out and Delete accounts fully works.

Polsihing UI on other pages.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
         android:allowBackup="true"
         android:dataExtractionRules="@xml/data_extraction_rules"
         android:fullBackupContent="@xml/backup_rules"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@drawable/my_digital_closet_logo"
```
Comment (NickZuke) [inline app/src/main/AndroidManifest.xml:11]: Should be resized to fit the dimensions of the app icon- low priority, I just really like that the icon shows up now truth be told.

**Review by awsone115 — APPROVED** [review]
I've reviewed your pull request and everything seems to be in order. You've done a nice job implementing the necessary changes, and I appreciate your attention to detail.

The android:allowBackup, android:dataExtractionRules, android:fullBackupContent, and android:icon attributes are all correctly set up. I don't see any need for changes at this point.

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (RomanWSU) [reply app/src/main/AndroidManifest.xml:11]: Yep totally agree. I will focus on it once I finish the other parts of the app.

Comment (Ayeshahelal) [reply app/src/main/AndroidManifest.xml:11]: The icon showing is definitely an accomplishment!

**Comment by Ayeshahelal** [comment]
Great job on fixing the code to allow this function to work properly! Everything looks great!

**Pull request merged**